client/src/Movies/SavedList.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-
+import { Link } from 'react-router-dom';
 export default class SavedList extends Component {
   constructor(props) {
     super(props);
```

Comment:
Be careful, this constructor is giving you a warning when your code is compiled. You could want to remove it.